zephyr/Kconfig

```diff
@@ -0,0 +1,46 @@
+if BOOTLOADER_MCUBOOT
+
+config MCUBOOT_CMAKELISTS_DIR
+	string "Path to the directory of the MCUBoot CMakeLists.txt file"
+	default "$MCUBOOT_BASE/boot/zephyr/"
+
+
+choice
+	prompt "MCUBoot build strategy"
+	default MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
+
+config MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE
+	# Mandatory option when being built through 'zephyr_add_executable'
+	bool "Use hex file instead of building MCUBoot"
+
+if MCUBOOT_BUILD_STRATEGY_USE_HEX_FILE
+
+config MCUBOOT_HEX_FILE
```

Comment:
I don't think you meant this to be a choice option; did you? Shouldn't it be moved after the `endchoice`?